packages/upset/src/components/AltTextSidebar.tsx

```diff
@@ -147,111 +114,136 @@ export const AltTextSidebar: FC<Props> = ({ open, close, generateAltText }) => {
+                    }
+                  }}
+                />
+              }
+              labelPlacement="start"
+            />
+            <HelpCircle 
+              text={"When enabled, allows you to enter a custom alternative text description."}
+              margin={{left: 12, top: 0, right: 0, bottom: 0}}
+            />
+            <br />
+            <FormControlLabel
+              style={{marginLeft: "5px"}} // Align with below text; has 2px border & 3px padding
+              sx={{ '& span': { fontSize: '0.8rem' } }} // Fontsize can't be set in style prop for some reason
+              label="Display Long Description"
+              control={
+                <Switch
+                  size="small"
+                  checked={useLong}
+                  tabIndex={8}
+                  onChange={(ev) => {
+                    setUseLong(ev.target.checked);
+                  }}
+                />
+              }
+              labelPlacement="start"
+            />
+            <HelpCircle 
+              text={"When enabled, displays the long text description for this plot instead of the short version."}
+              margin={{left: 12, top: 0, right: 0, bottom: 0}} 
+            />
+          </>) : (
+            <Typography variant="body1" color="error">{textGenErr}</Typography>
+          )}
+          {textEditing ? (<>
+            <Button 
+              color="primary" 
+              style={{float: 'right'}} 
+              onClick={() => {
+                setTextEditing(false);
+                if (!currState.useUserAlt)
+                  actions.setUseUserAltText(true);
+                if (currState.userAltText?.shortDescription !== userShortText 
+                    || currState.userAltText?.longDescription !== userLongText)  
+                  actions.setUserAltText({shortDescription: userShortText ?? "", longDescription: userLongText ?? ""});
+              }}
+              tabIndex={7}
+            >Save</Button>
+            <Button
+              color='warning'
+              style={{float: "right"}}
+              onClick={() => {
+                setUserLongText(altText?.longDescription);
+                setUserShortText(altText?.shortDescription);
+              }}
+              tabIndex={6}
+            >Reset Descriptions</Button>
+            <br />
+            <TextField multiline fullWidth
+              onChange={(e) => useLong ? setUserLongText(e.target.value) : setUserShortText(e.target.value)}
+              value={(displayAltText)}
+              tabIndex={5}
+            />
+            <br />
+          </>) : (
+            <div 
+              style={{
+                overflowY: 'auto',
+                cursor: 'pointer',
+                padding: '3px',
+                borderRadius: '4px',
+                width: 'calc(100% - 10px)', // We have 10px of padding + border
+                paddingBottom: '90px', // Necessary to keep it above the footer
```

Comment:
Is there a footer height constant anywhere?

Reply:
Not that I've found, but that should be added, make an issue?